src/client-python/reportclient/clientmodule.c

```diff
@@ -23,13 +23,13 @@
 static PyMethodDef module_methods[] = {
     /* method_name, func, flags, doc_string */
     /* for include/client.h */
-    { "alert"                     , p_alert                   , METH_VARARGS },
```

Comment:
If you’re going to change the style, do it separately.

Reply:
Ah, sorry, I did not want to change the coding style here. Just add the NULL at the end. I wrongly merged two WIP patches into this one (that is why also the docstrings were still present, I removed them in the latest versions)